packages/strapi-design-system/package.json

```diff
@@ -21,7 +21,7 @@
     "@floating-ui/react-dom": "^1.0.0",
     "@internationalized/number": "^3.1.1",
     "@radix-ui/react-use-callback-ref": "^1.0.0",
-    "@uiw/react-codemirror": "^4.19.1",
+    "@uiw/react-codemirror": "4.8.1",
```

Comment:
ℹ️ Following this suggestion https://github.com/uiwjs/react-codemirror/issues/216#issuecomment-1172859107, this is the only solution I have so far for getting the input to work in the CMS.

Reply:
Is that because we still have codemirror around in the CMS codebase or would that also happen if multiple JSON fields are used in one view?

Reply:
I don't think it is related to multiple codemirror packages in the codebase. I tested removing the v5 codemirror in the CMS and upgrading it to v6 but still had the problem. This is error is also coming from the scoped @codemirror and not codemirror. Good question about multiple fields in one view I'll try that.

Reply:
Mutliple fields in the same view seem to work fine